samples/react-sample-app/src/api/identity-verification-client.tsx

```diff
@@ -0,0 +1,107 @@
+import { default as authConfig } from "../config.json";
+import { AsgardeoSPAClient } from "@asgardeo/auth-react";
+
+const httpClient = AsgardeoSPAClient.getInstance();
+
+export const initiateVerification = async (): Promise<IdVResponseInterface> => {
+    return changeVerificationStatus(SdkFlowStatus.INITIATED)
+}
+
+export const completeVerification = async (): Promise<IdVResponseInterface> => {
+    return changeVerificationStatus(SdkFlowStatus.COMPLETED);
+}
+
+export const reinitiateVerification = async (): Promise<IdVResponseInterface> => {
+    return changeVerificationStatus(SdkFlowStatus.REINITIATED);
+}
+
+const changeVerificationStatus = async (status: SdkFlowStatus): Promise<IdVResponseInterface> => {
+
+    const requestConfig: HttpRequestConfig = {
+        url: `${authConfig?.baseUrl}/api/users/v1/me/idv/verify`,
+        method: IdVConstants.HTTP_POST,
+        headers: {
+            'Content-Type': IdVConstants.APPLICATION_JSON,
+            'Accept': IdVConstants.APPLICATION_JSON
+        },
+        data: {
+            "idVProviderId": authConfig?.identityVerificationProviderId,
+            "claims": [
+                "http://wso2.org/claims/dob",
+                "http://wso2.org/claims/givenname",
+                "http://wso2.org/claims/lastname"
+            ],
+            "properties": [
+                {
+                    "key": "status",
+                    "value": status
+                }
+            ]
+        }
+    }
+
+    return httpClient.httpRequest(requestConfig)
+        .then((response: HttpResponse<IdVResponseInterface>) => {
+            return response.data;
+        })
+        .catch((error) => {
+            throw error;
+        });
+}
+
+export const isClaimVerified = async (claimToVerify: string): Promise<ClaimVerificationStatus> => {
+
+    const requestConfig: HttpRequestConfig = {
+        url: `${authConfig?.baseUrl}/api/users/v1/me/idv/claims/`,
+        method: IdVConstants.HTTP_GET,
+        headers: {
+            'Content-Type': IdVConstants.URL_ENCODED,
+            'Accept': IdVConstants.APPLICATION_JSON
+        },
+        params: {
+            "idVProviderId": authConfig?.identityVerificationProviderId ?? "",
+        }
+    }
+
+    return httpClient.httpRequest(requestConfig)
+        .then((response: HttpResponse<IdVClaim[]>) => {
+            const claims = response.data as IdVClaim[];
+            const claim = claims.find((claim) => claim.uri === claimToVerify);
+            console.log("Found claim : ", claim );
+            return getClaimVerificationStatus(claim);
+        })
+        .catch((error) => {
+            throw error;
+        });
+}
+
+function getClaimVerificationStatus(claim: IdVClaim | undefined): ClaimVerificationStatus {
+    if (!claim) {
+        return { isVerified: undefined, workflowStatus: undefined };
+    }
+    return {
+        isVerified: claim.isVerified,
+        workflowStatus: claim.claimMetadata.onfido_workflow_status
+    };
+}
```

Comment:
1. Let's use the same convention used for previous functions
`export const getClaimVerificationStatus`

2. `undefined` is unnecessary in `claim: IdVClaim | undefined`
3. Function can be simplified with null check
```suggestion
function getClaimVerificationStatus(claim: IdVClaim | undefined): ClaimVerificationStatus {
    return {
        isVerified: claim?.isVerified,
        workflowStatus: claim?.claimMetadata?.onfido_workflow_status
    };
}
```

Reply:
```
if (!claim) {
        return { isVerified: undefined, workflowStatus: undefined };
    }
```
Above return value is mandatory for the claims on which the verification is not yet initiated.

The rest of the suggestion resolved at https://github.com/wso2-extensions/identity-verification-onfido/pull/17